src/pyuvdata/uvdata/mir_parser.py

```diff
@@ -4094,7 +4094,7 @@ def redoppler_data(
                 rx_code = np.median(self.bl_data["irec"][self.bl_data["ant1rx"] == 0])
                 rx_name = self.codes_data["rec"][rx_code]
                 if rx_name not in ("230", "345"):
-                    raise ValueError("Receiver code %i not recognized." % rx_code)
+                    raise ValueError(f"Receiver code {rx_code} not recognized.")
```

Comment:
I'm getting an error on this line because the old code cast it to an int and now it's a float. I can change it to cast to an int, but I wondered if we were sure that's what we wanted.

Reply:
@kartographer

Reply:
@bhazelton - I'd argue for casting it to an int for the sake of parsability (they're int16s in the underlying data and used as ints everywhere else, so the float printing may unintentionally confuse someone).